### PR TITLE
Merge branch 4.5 into master

### PR DESCRIPTION
**Pull request by DFolchA**
|Related issue|
|---|
||

## Description

This PR merges the latest changes in 4.5 into master.

### CODE REVIEW

**Review by verdx — CHANGES_REQUESTED** [review]
LGTM, small change needed

**Review by rauldpm — APPROVED** [review]
LGTM

File: unattended_installer/builder.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ readonly resources_certs="${base_path_builder}/cert_tool"
 readonly resources_passwords="${base_path_builder}/passwords_tool"
 readonly resources_common="${base_path_builder}/common_functions"
 readonly resources_download="${base_path_builder}/downloader"
-readonly source_branch="4.4"
+readonly source_branch="4.5"
```
[the remark this answers is not in the document]
Comment (jnasselle) [reply unattended_installer/builder.sh:19]: +1

[the remark this answers is not in the document]
Comment (DFolchA) [reply unattended_installer/builder.sh]: Done https://github.com/wazuh/wazuh-packages/pull/2123/commits/11a2a3e8ecc7d6251520568f1c330ad988a55b70

**Review by rauldpm — DISMISSED** [review]

**Review by verdx — APPROVED** [review]
LGTM

**Review by jnasselle — APPROVED** [review]
I've checked this PR in Jenkins and work as expected https://ci.wazuh.info/job/Packages_builder_tier/2906
LGTM!

**Review by rauldpm — COMMENTED** [review]
solaris/solaris10/pkginfo:VERSION="4.4.0"

**Review by rauldpm — CHANGES_REQUESTED** [review]
Review version references to older versions:

`solaris/solaris10/pkginfo:VERSION="4.4.0"`

**Review by rauldpm — APPROVED** [review]
LGTM

**Comment by jnasselle** [comment]
WIP
Checking this PR in Jenkins https://ci.wazuh.info/job/Packages_builder_tier/2914/
https://ci.wazuh.info/job/PR_wazuh/16694/

**Review by c-bordon — APPROVED** [review]
LGTM

**Review by rauldpm — APPROVED** [review]
LGTM

**Pull request merged**